app/src/gecko/java/org/mozilla/tv/firefox/utils/BuildFlavor.kt

```diff
@@ -42,4 +42,8 @@ class BuildFlavor {
     fun getInterceptionResponseContent(localizedContent: String): RequestInterceptor.InterceptionResponse.Content {
         return RequestInterceptor.InterceptionResponse.Content(localizedContent, "text/html", "base64")
     }
+
+    private fun generateGeckoVersionStr(): String {
+        return "${org.mozilla.geckoview.BuildConfig.MOZ_APP_VERSION}-${org.mozilla.geckoview.BuildConfig.MOZ_APP_BUILDID}"
```

Comment:
I don't know what MOZ_APP_VERSION is (like FF 68?) or MOZ_APP_BUILDID (e.g. like the random string of numbers specifying the build time) but this looks reasonable? Ideally, how their used should be specified in their kdoc but I guess that's hard because it's `BuildConfig`.

nit: in any case, I think it'd be helpful to add an inline comment above this line showing what this is supposed to look like, e.g.:
```kotlin
// Expected: 68.0a1-20190427
return "...